### PR TITLE
Feedback (PFT) - ACR report - Autumn 2023

### PR DESCRIPTION
**Pull request by duboisp**
ACR = Accessibility Conformance Report

This PR also include reference to accessibility assessment for feature-link and feedback.

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -965,8 +987,8 @@
 			},
 			"instructions": {
 				"en": [
-					"Ensure the <a href=\"https://github.com/wet-boew/GCWeb/releases\">version 13.9.0 or more recent of GCWeb</a> is implemented.",
-					"Ensure the <a href=\"../page-details/page-details-docs-en.html\">version 3.0 or more recent of the page details</a> is implemented.",
+					"Ensure the <a href=\"https://github.com/wet-boew/GCWeb/releases\">version 13.9.0 or newer of GCWeb</a> is implemented.",
+					"Ensure the <a href=\"../page-details/page-details-docs-en.html\">version 3.0 or newer of the page details</a> is implemented.",
```
Comment (duboisp) [inline _data/sites.json:991]: @Garneauma Can you confirm what text should be used? Thanks

File: _data/sites.json [hunk 2]
```diff
@@ -1121,8 +1143,8 @@
...
```
Comment (duboisp) [inline _data/sites.json:1147]: @Garneauma Can you confirm what text should be used? Thanks

Comment (Garneauma) [reply _data/sites.json:1147]: @duboisp "newer".

File: _data/sites.json (hunk 1 above)
Comment (duboisp) [reply _data/sites.json:991]: Confirmed, no change required.

File: _data/sites.json (hunk 2 above)
Comment (duboisp) [reply _data/sites.json:1147]: Confirmed, no change required.

File: _data/components.json [hunk 3]
```diff
@@ -121,6 +121,28 @@
...
 			}
+		],
+		"reports": [
+			{
+				"title": "Accessibility pre-assessment #1",
+				"language": "en",
+				"path": "reports/a11y-1-en.html"
+			},
+			{
+				"title": "Pré-assessment de l'accessibilité #1",
```
Comment (Garneauma) [inline _data/components.json:132]:
```suggestion
				"title": "Pré-évaluation de l'accessibilité #1",
```

File: _data/components.json [hunk 4]
```diff
@@ -121,6 +121,28 @@
...
+				"language": "fr",
+				"path": "reports/a11y-1-fr.html"
+			},
+			{
+				"title": "Accessibility assessment #1",
+				"language": "en",
+				"path": "reports/a11y-2-en.html"
+			},
+			{
+				"title": "Assessment de l'accessibilité #1",
```
Comment (Garneauma) [inline _data/components.json:142]:
```suggestion
				"title": "Évaluation de l'accessibilité #1",
```

File: _data/sites.json [hunk 5]
```diff
@@ -689,6 +689,28 @@
...
 			}
+		],
+		"reports": [
+			{
+				"title": "Accessibility assessment #1",
+				"language": "en",
+				"path": "reports/a11y-2-en.html"
+			},
+			{
+				"title": "Assessment de l'accessibilité #1",
```
Comment (Garneauma) [inline _data/sites.json:700]:
```suggestion
				"title": "Évaluation de l'accessibilité #1",
```

File: components/gc-featured-link/index.json-ld [hunk 6]
```diff
@@ -29,6 +29,28 @@
...
 			}
+		],
+		"reports": [
+			{
+				"title": "Accessibility pre-assessment #1",
+				"language": "en",
+				"path": "reports/a11y-1-en.html"
+			},
+			{
+				"title": "Pré-assessment de l'accessibilité #1",
```
Comment (Garneauma) [inline components/gc-featured-link/index.json-ld:40]:
```suggestion
				"title": "Pré-évaluation de l'accessibilité #1",
```

File: components/gc-featured-link/index.json-ld [hunk 7]
```diff
@@ -29,6 +29,28 @@
...
+				"language": "fr",
+				"path": "reports/a11y-1-fr.html"
+			},
+			{
+				"title": "Accessibility assessment #1",
+				"language": "en",
+				"path": "reports/a11y-2-en.html"
+			},
+			{
+				"title": "Assessment de l'accessibilité #1",
```
Comment (Garneauma) [inline components/gc-featured-link/index.json-ld:50]:
```suggestion
				"title": "Évaluation de l'accessibilité #1",
```

File: sites/feedback/index.json-ld [hunk 8]
```diff
@@ -62,6 +62,28 @@
...
 			}
+		],
+		"reports": [
+			{
+				"title": "Accessibility assessment #1",
+				"language": "en",
+				"path": "reports/a11y-2-en.html"
+			},
+			{
+				"title": "Assessment de l'accessibilité #1",
```
Comment (Garneauma) [inline sites/feedback/index.json-ld:73]:
```suggestion
				"title": "Évaluation de l'accessibilité #1",
```

File: sites/feedback/reports/acr-2023-autumn.json [hunk 9]
```diff
@@ -0,0 +1,733 @@
...
+      "https://wet-boew.github.io/GCWeb/sites/feedback/page-feedback-en.html?wbdisable=false",
+      "https://wet-boew.github.io/GCWeb/sites/feedback/page-feedback-en.html?wbdisable=true",
+      "https://wet-boew.github.io/GCWeb/sites/feedback/page-feedback-contact-en.html?wbdisable=false",
+      "https://wet-boew.github.io/GCWeb/sites/feedback/page-feedback-contact-en.html?wbdisable=true"
+    ],
+    "@type": [
+      "earl:TestSubject",
+      "schema:WebPage"
+    ],
+    "dct:description": "About the page feedback and its container section which is presented before the site footer and after the page content. It do include and are limited of the placement of the date modified component and the share this page component regarding how they are harmonizing with the page feedback tool. It do exclude the date modified component and the share this page component itself (ex. their inner HTML).",
```
Comment (Garneauma) [inline sites/feedback/reports/acr-2023-autumn.json:24]:
```suggestion
    "dct:description": "About the page feedback and its container section which is presented before the site footer and after the page content. It does include and is limited to the placement of the date modified component and the share this page component regarding how they are harmonizing with the page feedback tool. It excludes the date modified component and the share this page component by themselves (ex. their inner HTML).",
```

File: sites/feedback/reports/acr-2023-autumn.json [hunk 10]
```diff
@@ -0,0 +1,733 @@
...
+      "dct:description": "Confirm the score of 100%. There are no work items, and the numbering for WCAG 2.2 has changed since then, but I did not modify them. I removed the techniques identified per SC according to our new guidance for that property.",
+      "dct:issued": "2023-12-14",
+      "acr:assessment": "https://wet-boew.github.io/GCWeb/sites/feedback/reports/pre-a11y-2.json",
+      "acr:auditNote": [
+        {
+            "@type": "acr:AuditReportNote",
+            "acr:accuracy": "acr:falseNegative",
+            "dct:date": "2023-12-14",
+            "dct:title": "Standard mode - Accessible name of 'No' button",
+            "dct:description": "I do confirm its accessible name do match the text that is presented visually. That justification is not enough strong to make it fail the Success Criterion.",
```
Comment (Garneauma) [inline sites/feedback/reports/acr-2023-autumn.json:81]:
```suggestion
            "dct:description": "I confirm its accessible name does match the text that is presented visually. That justification is not strong enough to make it fail the Success Criterion.",
```

File: sites/feedback/reports/acr-2023-autumn.json [hunk 11]
```diff
@@ -0,0 +1,733 @@
...
+            "earl:test": "WCAG21:label-in-name",
+            "earl:outcome":"earl:passed",
+            "earl:subject": "_subject"
+        },
+        {
+            "@type": "acr:AuditReportNote",
+            "acr:accuracy": "acr:falseNegative",
+            "dct:date": "2023-12-14",
+            "dct:title": "Basic mode - Accessible name of 'No' button",
+            "dct:description": "I do confirm its accessible name do match the text that is presented visually. The button technical value used for form submission are disconnected and out of scope for the accessible name computation. That justification is not enough strong to make it fail the Success Criterion.",
```
Comment (Garneauma) [inline sites/feedback/reports/acr-2023-autumn.json:93]:
```suggestion
            "dct:description": "I confirm its accessible name does match the text that is presented visually. The button's technical value used for form submission is disconnected and out of scope for the accessible name computation. That justification is not strong enough to make it fail the Success Criterion.",
```

File: sites/feedback/reports/acr-2023-autumn.json [hunk 12]
```diff
@@ -0,0 +1,733 @@
...
+            "earl:test": "WCAG21:label-in-name",
+            "earl:outcome":"earl:passed",
+            "earl:subject": "_subject"
+        },
+        {
+            "@type": "acr:AuditReportNote",
+            "acr:accuracy": "acr:falseNegative",
+            "dct:date": "2023-12-14",
+            "dct:title": "Status messages about the character limit of the text field",
+            "dct:description": "The component are not displaying a status message about the number of character left so this Success Criteria is not applicable. Although, we can note the official attribute 'maxlength' of the text area element is used to report that content length limitation, which is done in conformance with the HTML5 specification, which is described as 'Maximum length of value', which is conforming to info and relationship Success Criteria 1.3.1. We can assume that limitation is calculable and predictable by the browser and/or the assistive technology. There is a risk where that HTML feature could be not fully reported to the user by some assistive technology. Further testing would be required and that are going to fall under improving the compatibility for specific assistive technology",
```
Comment (Garneauma) [inline sites/feedback/reports/acr-2023-autumn.json:105]:
```suggestion
            "dct:description": "The component is not displaying a status message about the number of characters left, so this Success Criterion is not applicable. Although, we can note that the official attribute 'maxlength' of the text area element is used to report that content length limitation, which is done in conformance with the HTML5 specification, which is described as 'Maximum length of value', which is conforming to info and relationship Success Criterion 1.3.1. We can assume that limitation is calculable and predictable by the browser and/or the assistive technology. There is a risk where that HTML feature could be not fully reported to the user by some assistive technology. Further testing would be required and that is going to fall under improving the compatibility for specific assistive technologies",
```

File: sites/feedback/reports/acr-2023-autumn.json [hunk 13]
```diff
@@ -0,0 +1,733 @@
...
+        }
+      ]
+    }
+  ],
+
+  "wf:task": [
+    {
+      "@type": "wf:Task",
+      "dct:title": "Compatibility research on how AT are supporting the maxlength attribute",
+      "wf:goalDescription": "Document various AT and browser compatibility test regarding how they are clearing announcing the input content limitation set through the attribute 'maxlength' for a textaria element. Goal is to propose, if issue are found, solution that could improve the user experience to ensure they are advised when they are informed, reaching out or be close to reach that max length limit.",
```
Comment (Garneauma) [inline sites/feedback/reports/acr-2023-autumn.json:120]:
```suggestion
      "wf:goalDescription": "Document various AT and browser compatibility tests regarding how they are clearing announcing the input content limitation set through the attribute 'maxlength' for a textaria element. Goal is to propose, if issues are found, a solution that could improve the user experience to ensure they are advised when they have reached or close to reach that max length limit.",
```

**Review by Garneauma — CHANGES_REQUESTED** [review]

**Review by Garneauma — APPROVED** [review]

**Pull request merged**